docs/backup-restore.md

```diff
@@ -1,134 +1,208 @@
+</div>
+<div id="Cosmos_DB_for_NoSQL2" class="tabcontent" markdown="1">
 
-To avoid mistakes, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
-Cassy is also integrated with `scalar-admin` so it can issue a pause request to the ScalarDB application (or ScalarDB Server) of a Cassandra cluster.
-Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+You must create a Cosmos DB for NoSQL account with a continuous backup policy that has the PITR feature enabled. After enabling the feature, backups are created continuously.
 
-**Cosmos DB for NoSQL**
+To specify a transactionally consistent restore point, pause your application that is using ScalarDB with Cosmos DB for NoSQL as described in [Back up with explicit pausing](#back-up-with-explicit-pausing).
+</div>
+<div id="DynamoDB2" class="tabcontent" markdown="1">
 
-You must create a Cosmos DB for NoSQL account with a continuous backup policy enabled to use point-in-time restore (PITR) feature. Backups are created continuously after it is enabled.
-To specify a transactionally-consistent restore point, please pause the ScalarDB application of a Cosmos DB for NoSQL as described [here](#transactionally-consistent-backups-during-pause-duration).
+You must enable the PITR feature for DynamoDB tables. If you're using [ScalarDB Schema Loader](schema-loader.md) to create schemas, the tool enables the PITR feature for tables by default.
 
-**DynamoDB**
+To specify a transactionally consistent restore point, pause your application that is using ScalarDB with DynamoDB as described in [Back up with explicit pausing](#back-up-with-explicit-pausing).
+</div>
+</div>
 
-You must enable the point-in-time recovery (PITR) feature for DynamoDB tables. If you use [ScalarDB Schema Loader](schema-loader.md) for creating schema, it enables PITR feature for tables by default.
-To specify a transactionally-consistent restore point, please pause the ScalarDB application of a DynamoDB as described [here](#transactionally-consistent-backups-during-pause-duration).
+## Restore a backup
 
-## Restore Backup
+How you restore a transactionally consistent backup depends on the type of database that you're using. Select a database to see how to create a transactionally consistent backup for ScalarDB.
 
-### MySQL (restore)
+{% capture notice--warning %}
+**Note**
 
-If you use MySQL and `mysqldump` to create a backup file, use `mysql` command to restore the file as specified [in the MySQL doc](https://dev.mysql.com/doc/mysql-backup-excerpt/8.0/en/reloading-sql-format-dumps.html).
+The backup methods by database listed below are just examples of some of the databases that ScalarDB supports.
```

Comment:
```suggestion
The restore methods by database listed below are just examples of some of the databases that ScalarDB supports.
```
It seems that the `restore` is correct here.

Reply:
Ah, thanks for catching that!